imagefit/views.py

```diff
@@ -41,7 +41,7 @@ def resize(request, path_name, format, url):
     statobj = os.stat(image.path)
     if not was_modified_since(request.META.get('HTTP_IF_MODIFIED_SINCE'),
                               statobj[stat.ST_MTIME], statobj[stat.ST_SIZE]):
-        return HttpResponseNotModified(mimetype=image.mimetype)
+        return HttpResponseNotModified() #mimetype=image.mimetype)
```

Comment:
Could you remove the comment ` #mimetype=image.mimetype)` here?
If we get rid of it let's just clear it out